core/tooltip.ts

```diff
@@ -19,7 +19,7 @@ import * as blocklyString from './utils/string.js';
 export type TipInfo =
   | string
   | {tooltip: AnyDuringMigration}
-  | (() => TipInfo | string | Function);
+  | (() => TipInfo | string);
```

Comment:
This is also a breaking change that prevents developers from using [provided functionality](https://github.com/google/blockly/blob/ec8f9c8589fd0ea849eba4ae80f59cb878a0f24a/core/tooltip.ts#L150).

Reply:
Ah this actually is breaking. I read the signature incorrectly. Thank you for the catch, I'll fix tomorrow.

Reply:
Actually nvm, I think I was right the first time. Removing the complexity with the object, this simplifies to:

```ts
type TipeInfo = string | (() => TipInfo);
```

Which is properly recursively defined so you can return a function from the function. We could remove the `| string` from the function for clarity. But I don't think it's necessary.

Reply:
> Actually nvm, I think I was right the first time. Removing the complexity with the object, this simplifies to:
> 
> ```ts
> type TipeInfo = string | (() => TipInfo);
> ```
> 
> Which is properly recursively defined so you can return a function from the function. We could remove the `| string` from the return type of the function for clarity. But I don't think it's necessary.

Oh, you are absolutely correct about this.  My apologies—though I am reassured that my error was at least not _instantly_ obvious…